updatecli/values.yaml

```diff
@@ -7,3 +7,6 @@ github:
   branch: "main"
   owner: "jenkins-infra"
   repository: "docker-packaging"
+
+jdk:
+  majorVersion: 17
```

Comment:
Optional suggestion: WDYT about putting this value in https://github.com/jenkins-infra/docker-packaging/blob/main/Makefile?

Reply:
What would be the goal of specifying it in the `Makefile`? Not sure of the goal (and how it would work)

Reply:
Or using

> Added a target to update the major JDK in the test harness (cst.yml)

Maybe not applicable?

Reply:
I don't understand what you are asking?

Reply:
Discussed together, this version concerns only updatecli, its current location makes sense.